CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## 2.3.2 (2021-01-18)
```

Comment:
https://github.com/schubergphilis/terraform-aws-mcaf-avm/pull/36 got merged a few hours ago, can you rebase and version this 2.5.0?

Reply:
Done